ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -985,7 +985,7 @@ Loader {
                         senderColorId: Utils.colorIdForPubkey(root.senderId)
                         paymentRequestModel: root.paymentRequestModel
                         playAnimations: root.Window.active && root.messageStore.isChatActive
-                        isOnline: root.rootStore.isOnline
```

Comment:
This is a stores refactoring error; the (chat) `rootStore` doesn't have that property